src/ops/StateOps.ts

```diff
@@ -30,4 +36,56 @@ export default class StateOps extends BaseOp {
             });
         })
     }
-}
\ No newline at end of file
+
+    queryStates(requestObject: QueryHistoryStatesRequest): Promise<HistoryStateResults> {
+        return new Promise<HistoryStateResults>((resolve, reject)=>{
+            var contentType: string;
+            var url = this.baseUrl;
+            if(requestObject.traitAlias){
+                contentType = "application/vnd.kii.TraitStateQueryRequest+json";
```

Comment:
should check whether requestObject.traitAlias is not empty string if it is not null.